CHANGELOG.md

```diff
@@ -595,37 +550,6 @@ releases.
     `example/cumulus-tf/sync_granule_workflow.asl.json` to include `ACL`
     parameter.
 
-## [v11.1.8] 2022-11-07 [BACKPORT]
```

Comment:
I'm confused.  Why is this deleted?  Any thing under ## [v13.4.0] 2022-10-31 shouldn't be changed